### PR TITLE
Community web based Form integration

### PR DESCRIPTION
**Pull request by vinayaksh42**
**Description**
Community web based form within layer5.io using integromat

**Notes for Reviewers**
Form integration with Integromat

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b1ef9d89754811baebe72dff2d19a5592182a70a at: https://60f0ac01620cf710a6b6bc91--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8e3b5719157e0b8546d4dcd29e8b284dbf95f2cf at: https://60f0afb01017601148a3f683--layer5ng.netlify.app

**Comment by vinayaksh42** [comment]
https://60f0afb01017601148a3f683--layer5ng.netlify.app/community/webform 
link for the web form page

**Comment by leecalcote** [comment]
Oh, we simplified the flow to be the same for all types of members irrespective of why they are here. There is a single path for all members to follow as they fill in the form.

**Comment by leecalcote** [comment]
Link to form flow - https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/?node-id=5363%3A0

**Comment by vinayaksh42** [comment]
okay, Will make changes accordingly to the Form

**Comment by leecalcote** [comment]
Heads-up on a responsive issue on this step.

Screen Shot 2021-07-15 at 6 25 59 PM

**Comment by l5io** [comment]
🚀 Preview for commit fbd6c9f8d1a8ecc12a4b5e6bc0307b6c76e22a70 at: https://60f0c607785d452b1474f971--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5c91634ca0c668446e82b854647bcb3215527f8d at: https://60f0ce1240044935b4c58439--layer5ng.netlify.app

**Comment by vinayaksh42** [comment]
> Heads-up on a responsive issue on this step.
> 
> Screen Shot 2021-07-15 at 6 25 59 PM

There are a few mobile view issues, I will focus on them as soon as the integromat is properly setup.

**Comment by l5io** [comment]
🚀 Preview for commit b55a9068797c5c64a1c082653f969b01d0a940ae at: https://60f1a3d09ecb46373e551a01--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 01dd0c8b295ade54d8c03d78af11c04ac4deefe4 at: https://60f2f54e38f74a19ba48cec5--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 335faca8080082351baa9f7b5d5e6ecbd02057d2 at: https://60f3e1f053b7842b6f19f27f--layer5ng.netlify.app

**Review by leecalcote — APPROVED** [review]

**Comment by Nikhil-Ladha** [comment]
@vinayaksh42 is the form ready and working or is there something still left?

**Comment by vinayaksh42** [comment]
> @vinayaksh42 is the form ready and working or is there something still left?

It still requires integromat setup. I am working on it

**Comment by leecalcote** [comment]
In the meantime, I think we're ok to merge this. When it's ready to be used, let's update it's location to layer5.io/newcomer.

**Comment by Nikhil-Ladha** [comment]
> In the meantime, I think we're ok to merge this. When it's ready to be used, let's update it's location to layer5.io/newcomer.

As @vinayaksh42 says. I am fine with both.

**Comment by vinayaksh42** [comment]
> In the meantime, I think we're ok to merge this. When it's ready to be used, let's update it's location to layer5.io/newcomer.

Okay I will change it to /newcomers

File: src/sections/Community/Web-based-from/index.js [hunk 1]
```diff
@@ -92,68 +75,106 @@ const WebBasedForm = () => {
...
+                <Field type="radio" name="tshirtSize" value="L" />
+                L
+              </label>
+              <label>
+                <Field type="radio" name="tshirtSize" value="XL" />
+                XL
+              </label>
+              <label>
+                <Field type="radio" name="tshirtSize" value="XXL" />
+                XXL
```
[the remark this answers is not in the document]
Comment (vinayaksh42) [reply src/sections/Community/Web-based-from/index.js:138]: sure

**Comment by vinayaksh42** [comment]
@Nikhil-Ladha @leecalcote we are good to go with the above commits. I have added the required changes 

- change of route to /newcomers 
- spacing in radio buttons
- active state border color

**Comment by vinayaksh42** [comment]
Next up is configuring the integromat in the same way we have the google form

**Comment by l5io** [comment]
🚀 Preview for commit e09ed6127614aff5064724e1bcf0ce41c3ade3b6 at: https://60feac638ae5ef24931338b0--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Those two endpoints are so similar. It's easy to get them confused. Careful: one of them currently works and the other one doesn't.

**Comment by leecalcote** [comment]
This a good note to send to the #community-managers channel or @layer5.io mailer (using the same alias, but without the # sign).

**Pull request merged**